spec/semantic_range_spec.rb

```diff
@@ -61,13 +61,13 @@
       v0 = tuple[0]
       v1 = tuple[1]
       loose = tuple[2]
-      expect(SemanticRange.gt(v0, v1, loose)).to eq(true)
```

Comment:
To ensure we don't break compatibility, can we keep testing the ol dmethod names at least for a couple of things? I know there's the explicit "is this aliased" check below but it's probably also worth ensuring nothing weird happens

Reply:
sounds good! 👍